definitions/apm-application/golden_metrics.yml

```diff
@@ -3,16 +3,16 @@ responseTimeMs:
   query:
     select: average(newrelic.timeslice.value) * 1000 AS 'Response time (ms)'
     where: metricTimesliceName in ('HttpDispatcher', 'OtherTransaction/all')
-    facet: appName
+    eventName: appName
```

Comment:
Is this really an event name? (e.g. do events have a name?)

Reply:
this is an eventName as envetId is an id of the event :)

don't love the names of those but we keep consistency since they are exposed in some apis